pii_recognition/experiments/pii_validation/comprehend_pii.yaml

```diff
@@ -0,0 +1,76 @@
+benchmark_data_file: pii_recognition/datasets/predisio_fake_pii/generated_size_500_date_August_25_2020.json
+recogniser_name: ComprehendRecogniser
+recogniser_params:
+  supported_entities:
+    - BANK_ACCOUNT_NUMBER
+    - BANK_ROUTING
+    - CREDIT_DEBIT_NUMBER
+    - CREDIT_DEBIT_CVV
+    - CREDIT_DEBIT_EXPIRY
+    - PIN
+    - NAME
+    - ADDRESS
+    - PHONE
+    - EMAIL
+    - AGE
+    - USERNAME
+    - PASSWORD
+    - URL
+    - AWS_ACCESS_KEY
+    - AWS_SECRET_KEY
+    - IP_ADDRESS
+    - MAC_ADDRESS
+    - SSN
+    - PASSPORT_NUMBER
+    - DRIVER_ID
+    - DATE_TIME
+  supported_languages: 
+    - en
+  model_name: pii
+grouped_targeted_labels:
+  - 
+    - CREDIT_CARD
+    - BANK_ACCOUNT_NUMBER
+    - BANK_ROUTING
+    - CREDIT_DEBIT_NUMBER
+    - CREDIT_DEBIT_CVV
+    - CREDIT_DEBIT_EXPIRY
+    - PIN
+  - 
+    - US_SSN
+    - SSN
+  -
+    - URL
+  - 
+    - IP_ADDRESS
+    - MAC_ADDRESS
+  - 
+    - PERSON
+  -
+    - EMAIL
+  -
+    - BIRTHDAY
+    - DATE_TIME
+  -
+    - PHONE_NUMBER
+    - PHONE
+  -
+    - LOCATION
+    - ADDRESS
+  -
+    - PERSON
+    - USERNAME
+    - NAME
+nontargeted_labels:
+  - NATIONALITY
+  - IBAN
+  - TITLE
+  - ORGANIZATION
+  - AGE
+  - AWS_SECRET_KEY
+  - AWS_ACCESS_KEY
+  - PASSWORD
+  - PASSPORT_NUMBER
+  - DRIVER_ID
```

Comment:
Remove those entities because benchmark data does not have those labels included.